.github/workflows/run-e2e.yml

```diff
@@ -58,7 +57,7 @@ jobs:
           publish_branch: gh-pages/dimpact-e2e-test-report # deploying to gh-pages branch
           keep_files: true # retains files that are not part of the current publish
       - name: Notify Slack
-        if: github.ref == 'refs/heads/main'
+        if: always()
```

Comment:
I don't think we want this. We discussed this I think before your time and we decided to only notify Slack from the main branch. Otherwise people will run the e2e tests from their branch, which is fine, but then other developers will see the Slack notifications and get very confused. We don't want that I think. Better to do Slack notifications only from the main branch I think.

Reply:
The `if` is never executed on previous step error
Unless ... there's always()

Reply:
Ok. The expression is not evaluated if there are no status checks too. The latest version works around this by adding `! cancelled()`